docs/user-flows.md

```diff
@@ -1,262 +1,157 @@
 # User Flows in Lighthouse
 
-> Motivation: You want to run Lighthouse on your whole site, not just the landing page.
+Historically, Lighthouse has analyzed the cold pageload of a page. Starting in 2022 (Lighthouse v10), it can analyze and report on the entire page lifecycle via "user flows".
 
-> Is that checkout form after you add an item to your cart accessible?
+#### You might be interested in flows if…
 
-> What is the Cumulative Layout Shift of my SPA page transition?
+* … you want to run Lighthouse on your whole webapp, not just the landing page.
+* … you want to know if the checkout process is accessible.
+* … you want to know the Cumulative Layout Shift of my SPA page transition
 
-You want Lighthouse on a _flow_, not just a page load.
+In these cases, you want Lighthouse on a _flow_, not just a page load.
 
-This document describes how Lighthouse measures flows and offers recommendations on how to structure your own flow measurement with practical examples.
+## The three modes: Navigation, Timespan, Snapshot
 
-## Flow Building Blocks
+Lighthouse can now run in three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
 
-Flow measurement in Lighthouse is enabled by three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
+* <img src="https://user-images.githubusercontent.com/39191/168928225-f2157fda-5131-4bd0-9121-b1a0b2f869a7.png" height="80" align="middle"> **Navigation mode** analyzes a single page load. Prior to v10, all Lighthouse runs were essentially in this mode.
```

Comment:
IMO these images would be better to visually identify the headings below. I know I'm in the user flow section cuz I saw the user flow image, so maybe that's how these images should be used.

I also think they look kinda weird as part of a bulleted list.

Reply:
Yeah. you'll see in that screenshot above i experimented with imagery in the table.
agree the inline image on bullet points is kinda whack

ill try this out.. i think it'll work out well. :)